# generated by patch-package 6.4.8 on 2022-02-15 21:23:01
#
# command:
#   npx patch-package antd-mobile
#
# declared package:
#   antd-mobile: ^5.0.0-rc.27
#
diff --git a/node_modules/antd-mobile/2x/cjs/components/mask/mask.d.ts b/node_modules/antd-mobile/2x/cjs/components/mask/mask.d.ts
index 6b375f6..0b3d029 100644
--- a/node_modules/antd-mobile/2x/cjs/components/mask/mask.d.ts
+++ b/node_modules/antd-mobile/2x/cjs/components/mask/mask.d.ts
@@ -14,5 +14,6 @@ export declare type MaskProps = {
     afterShow?: () => void;
     afterClose?: () => void;
     stopPropagation?: PropagationEvent[];
+    dissipationDuration?: number;
 } & NativeProps<'--z-index'>;
 export declare const Mask: React.FC<MaskProps>;
diff --git a/node_modules/antd-mobile/2x/es/components/mask/mask.d.ts b/node_modules/antd-mobile/2x/es/components/mask/mask.d.ts
index 6b375f6..0b3d029 100644
--- a/node_modules/antd-mobile/2x/es/components/mask/mask.d.ts
+++ b/node_modules/antd-mobile/2x/es/components/mask/mask.d.ts
@@ -14,5 +14,6 @@ export declare type MaskProps = {
     afterShow?: () => void;
     afterClose?: () => void;
     stopPropagation?: PropagationEvent[];
+    dissipationDuration?: number;
 } & NativeProps<'--z-index'>;
 export declare const Mask: React.FC<MaskProps>;
diff --git a/node_modules/antd-mobile/2x/es/components/mask/mask.js b/node_modules/antd-mobile/2x/es/components/mask/mask.js
index d09e664..5455644 100644
--- a/node_modules/antd-mobile/2x/es/components/mask/mask.js
+++ b/node_modules/antd-mobile/2x/es/components/mask/mask.js
@@ -49,7 +49,8 @@ export const Mask = p => {
       mass: 1,
       tension: 200,
       friction: 30,
-      clamp: true
+      clamp: true,
+      duration: props.dissipationDuration || 150,
     },
     onStart: () => {
       setActive(true);
diff --git a/node_modules/antd-mobile/cjs/components/mask/mask.d.ts b/node_modules/antd-mobile/cjs/components/mask/mask.d.ts
index 6b375f6..0b3d029 100644
--- a/node_modules/antd-mobile/cjs/components/mask/mask.d.ts
+++ b/node_modules/antd-mobile/cjs/components/mask/mask.d.ts
@@ -14,5 +14,6 @@ export declare type MaskProps = {
     afterShow?: () => void;
     afterClose?: () => void;
     stopPropagation?: PropagationEvent[];
+    dissipationDuration?: number;
 } & NativeProps<'--z-index'>;
 export declare const Mask: React.FC<MaskProps>;
diff --git a/node_modules/antd-mobile/es/components/mask/mask.d.ts b/node_modules/antd-mobile/es/components/mask/mask.d.ts
index 6b375f6..0b3d029 100644
--- a/node_modules/antd-mobile/es/components/mask/mask.d.ts
+++ b/node_modules/antd-mobile/es/components/mask/mask.d.ts
@@ -14,5 +14,6 @@ export declare type MaskProps = {
     afterShow?: () => void;
     afterClose?: () => void;
     stopPropagation?: PropagationEvent[];
+    dissipationDuration?: number;
 } & NativeProps<'--z-index'>;
 export declare const Mask: React.FC<MaskProps>;
diff --git a/node_modules/antd-mobile/es/components/mask/mask.js b/node_modules/antd-mobile/es/components/mask/mask.js
index d09e664..5455644 100644
--- a/node_modules/antd-mobile/es/components/mask/mask.js
+++ b/node_modules/antd-mobile/es/components/mask/mask.js
@@ -49,7 +49,8 @@ export const Mask = p => {
       mass: 1,
       tension: 200,
       friction: 30,
-      clamp: true
+      clamp: true,
+      duration: props.dissipationDuration || 150,
     },
     onStart: () => {
       setActive(true);
